pkgs/development/tools/misc/semver-tool/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "semver-tool-${version}";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "fsaintjacques";
+    repo = "semver-tool";
+    rev = version;
+    sha256 = "0lpwsa86qb5w6vbnsn892nb3qpxxx9ifxch8pw3ahzx2dqhdgnrr";
+  };
+
+  buildPhase = "true"; # otherwise we try to 'make' which fails.
+
+  installPhase = ''
```

Comment:
Why was https://github.com/NixOS/nixpkgs/pull/51004/files#r236070972 removed?

Reply:
What was removed? The link yields nothing different than my patchset.

Reply:
Ahh, that should have been a link to a review comment to use

```
runHook preInstall
```

and
```
runHook postInstall
```

At the beginnings and ends of the `installPhase`.

Reply:
I guess because I cherry-picked another commit, didn't I? Fixed with fixup commit.

Reply:
Yep that is what happened https://github.com/NixOS/nixpkgs/pull/51004/commits/240e5c3c1f54f5f84a759298e390b1a51eaad72c